Library/Homebrew/extend/ENV.rb

```diff
@@ -33,19 +33,30 @@ def activate_extensions!(env: nil)
 
   sig {
     params(
-      env:          T.nilable(String),
-      cc:           T.nilable(String),
-      build_bottle: T::Boolean,
-      bottle_arch:  T.nilable(String),
-      _block:       T.proc.returns(T.untyped),
+      env:           T.nilable(String),
+      cc:            T.nilable(String),
+      build_bottle:  T::Boolean,
+      bottle_arch:   T.nilable(String),
+      debug_symbols: T.nilable(T::Boolean),
+      _block:        T.proc.returns(T.untyped),
     ).returns(T.untyped)
   }
-  def with_build_environment(env: nil, cc: nil, build_bottle: false, bottle_arch: nil, &_block)
+  def with_build_environment(
+    env: nil,
+    cc: nil,
+    build_bottle: false,
+    bottle_arch: nil,
+    debug_symbols: false,
+    &_block
+  )
     old_env = to_hash.dup
     tmp_env = to_hash.dup.extend(EnvActivation)
     T.cast(tmp_env, EnvActivation).activate_extensions!(env: env)
     T.cast(tmp_env, T.any(Superenv, Stdenv))
-     .setup_build_environment(cc: cc, build_bottle: build_bottle, bottle_arch: bottle_arch)
+     .setup_build_environment(
+       cc: cc, build_bottle: build_bottle, bottle_arch: bottle_arch,
```

Comment:
May as well split each parameter to a new line and add a trailing comma on each (and same to all similar cases below)

Reply:
Done in cd9fe97c551d4e9b5f0a702a1382a6ccbe494773

Reply:
Consistently made all calls as short as possible (might have been the wrong thing but there were quite a few...) in cd9fe97c551d4e9b5f0a702a1382a6ccbe494773